src/main/kotlin/no/nav/syfo/behandlendeenhet/database/NavUtlandQuery.kt

```diff
@@ -0,0 +1,75 @@
+package no.nav.syfo.behandlendeenhet.database
+
+import no.nav.syfo.application.database.DatabaseInterface
+import no.nav.syfo.application.database.toList
+import no.nav.syfo.behandlendeenhet.database.domain.PNavUtland
+import no.nav.syfo.domain.PersonIdentNumber
+import java.sql.*
+import java.time.OffsetDateTime
+import java.util.*
+
+const val queryUpdateNavUtland =
+    """
+        INSERT INTO NAV_UTLAND (
```

Comment:
Dette er jo ganske stilig, da blir det vel en atomisk operasjon?

Reply:
Ser sånn ut 🙌🏼 
https://wiki.postgresql.org/wiki/UPSERT

> Primary goals:
In general, guarantee insert-or-update "atomicity" for the simple cases - guarantee one or the other of those two outcomes ("the essential property of UPSERT").